packages/flutter/lib/src/material/carousel.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:math' as math;
 
+import 'package:collection/collection.dart';
```

Comment:
Wondering: What are you using from this package?

Reply:
I think it's the `max` getter from a List, like `weights.max` in this file